x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/schemas/types/lists_default_array.test.ts

```diff
@@ -70,7 +172,10 @@ describe('lists_default_array', () => {
     const decoded = ListsDefaultArray.decode(payload);
     const message = pipe(decoded, foldLeftRight);
 
-    expect(getPaths(left(message.errors))).toEqual(['Invalid value "5" supplied to ""']);
+    expect(getPaths(left(message.errors))).toEqual([
+      'Invalid value "5" supplied to ""',
```

Comment:
@FrankHassanabad @dhurley14 I wasn't able to figure out why this shows up twice here. I noticed that when you use t.recursion or t.union, it seems to explore both paths, but not sure why it's happening here.